database/models/users.db.php

```diff
@@ -132,7 +136,7 @@ static function getCurrent(): User|null
         if (!has_session("user_id")) {
             return null;
         }
-        if (isset($_SESSION['controlled_user_id'])) {
+        if (isset ($_SESSION['controlled_user_id'])) {
```

Comment:
same here. What formatter do you use?

Reply:
I use PSR-12 code formatter, with PHP extension of devsense.
I checked, and `isset` is actually not a normal function, but a language construct that is used like a function. Hence the white space added by the formatter like you would add one for an `if` 🤯